### PR TITLE
[Clean fluid] Clean fluid resize_linear and image_resize_short.

### PR DESCRIPTION
**Pull request by yunyaoXYY**
### PR types
Others 

### PR changes
Others

### Describe
Remove fluid API and related unittests.
In the nn.py, these APIs are:
resize_linear
image_resize_short

### CODE REVIEW

**Review by jiangjiajun — DISMISSED** [review]
LGTM

**Review by Ligoml — DISMISSED** [review]
LGTM for docs

File: python/paddle/fluid/layers/nn.py [hunk 1]
```diff
@@ -101,9 +101,6 @@
     'pad',
     'roi_pool',
     'roi_align',
-    'image_resize',
```
Comment (jeff41404) [inline python/paddle/fluid/layers/nn.py:104]: shall we also delete `def image_resize` in near L7500？

Comment (yunyaoXYY) [reply python/paddle/fluid/layers/nn.py:104]: The function of "def image_resize" will be deleted in later commits.

Comment (yunyaoXYY) [reply python/paddle/fluid/layers/nn.py:104]: > shall we also delete `def image_resize` in near L7500？

The function of "def image_resize" will be deleted in later commits.

Comment (yunyaoXYY) [reply python/paddle/fluid/layers/nn.py:104]: Done. Add image_resize back to __all__ list.

**Review by jeff41404 — APPROVED** [review]
LGTM

**Review by Ligoml — APPROVED** [review]

**Review by XiaoguangHu01 — APPROVED** [review]
LGTM

**Pull request merged**